openml/tasks/task.py

```diff
@@ -36,7 +49,7 @@ class OpenMLTask(OpenMLBase):
     def __init__(
         self,
         task_id: Optional[int],
-        task_type_id: int,
+        task_type_id: TaskType,
```

Comment:
And a rename to `task_type` here as well.

Reply:
Hey, there is a str `task_type` already being passed to this function. Should I change its name?

Reply:
Ah. I am having some troubles right now with figuring out what the effect of that change will be. It's fine to leave it like it is, I'll have a look at it myself later 👍